src/scripts.ts

```diff
@@ -0,0 +1,161 @@
+import * as _ from "lodash";
+import * as fs from "fs";
+import {EventEmitter} from "events";
+import * as log4js from "log4js";
+import * as Linter from "tslint";
+import {LintResult} from "tslint/lib/lint";
+import {Server as KarmaServer, ConfigOptions as KarmaOptions} from "karma";
+
+import * as utils from "./utils";
+import config from "./config";
+
+interface KarmaConfig extends KarmaOptions {
+  jspm: any;
+}
+
+export class ScriptLinter extends EventEmitter {
+
+  private configuration = JSON.parse(fs.readFileSync("tslint.json", "utf8"));
```

Comment:
It should be possible to provide overrides for the contents of the file, or to overwrite the whole file path.